features/et1/et1_apply_page_in_welsh_translation.feature

```diff
@@ -0,0 +1,26 @@
+@e2e
+@javascript
+Feature:
+  As a claimant
+  I would like to select whether I submit my application in Welsh or English
+  So that I can use my preferred language
+
+  Background: Apply Page in Welsh
+    Given a claimant is on the Apply page in 'Welsh' 
+
+  Scenario: Your feedback
```

Comment:
I think this will be slow as each scenario will be fetching the page again - and whilst the apply page is the first page, when you get round to testing something which is say 6 pages in, you wont want to be doing separate scenarios for different bits of content like this ?